tests/30rooms/13guestaccess.pl

```diff
@@ -143,11 +143,14 @@
       })->then( sub {
          matrix_get_room_membership( $user, $room_id, $guest_user );
```

Comment:
I think these lines are these now redundant(?)

Reply:
Yep, this is redundant. Removed :)